# Request
string aim_type  
bool if_parking
---
# Response
bool if_success
string obj_name
